RoundsWithFriends/GameModes/RWFGameModeHandler.cs

```diff
@@ -28,7 +31,10 @@ public RWFGameModeHandler(string name, string gameModeId, bool allowTeams, int p
                 { "playersRequiredToStartGame", UnityEngine.Mathf.Clamp(playersRequiredToStartGame ?? 2, 1, RWFMod.MaxPlayersHardLimit) },
                 { "maxPlayers", UnityEngine.Mathf.Clamp(maxPlayers ?? RWFMod.instance.MaxPlayers, 1, RWFMod.MaxPlayersHardLimit) },
                 { "maxTeams", UnityEngine.Mathf.Clamp(maxTeams ?? RWFMod.instance.MaxTeams, 1, RWFMod.MaxColorsHardLimit) },
-                { "maxClients", UnityEngine.Mathf.Clamp(maxClients ?? RWFMod.instance.MaxClients, 1, RWFMod.MaxPlayersHardLimit) }
+                { "maxClients", UnityEngine.Mathf.Clamp(maxClients ?? RWFMod.instance.MaxClients, 1, RWFMod.MaxPlayersHardLimit) },
+                { "description", description ?? ""},
+                { "descriptionFontSize", descriptionFontSize ?? 30},
+                { "videoURL", videoURL ?? "https://media.giphy.com/media/50dtBlALJ5jIgmnasA/giphy.mp4"}
```

Comment:
Would it make more sense to add these as properties to `RWFGameModeHandler`? Semantically `Settings` is meant for game settings, which these are not.

Reply:
side note: this would require that you test if a game mode handler inherits `RWFGameModeHandler`. I don't see this as a negative since you would need to test for existence of these variables/settings anyway.

Reply:
Added `UISettings` to Unbound's `IGameModeHandler` and updated this PR accordingly